js/packages/teams-ai/src/AI.ts

```diff
@@ -18,7 +18,6 @@ import { TurnState } from './TurnState';
 
 /**
  * A function that can be used to select a prompt to use for the current turn.
- *
  * @template TState Type of the turn state.
```

Comment:
:o jsdocs should make up its mind. we just added these :/

Reply:
ya I think this is because I was able to update alot of the package versions during this process 🤷‍♂️

Reply:
🙃 Resolving - just me complaining